src/checkers/firefoxchecker.py

```diff
@@ -39,44 +39,50 @@
 
 
 class FirefoxChecker(Checker):
-    FIREFOX_RELEASE_INFO_URL = 'https://product-details.mozilla.org/1.0/firefox_versions.json'  # noqa: E501
-    FIREFOX_ARCHIVE_BASE_URL = 'https://archive.mozilla.org/pub/firefox/releases/{version}/'  # noqa: E501
-    FIREFOX_PLATFORM = 'linux-x86_64'
-    CHECKER_BROWSER_SOURCE_FILENAME = 'firefox.tar.bz2'
-    CHECKER_BROWSER_FILENAME_MATCH = r'firefox-.+\.tar\.bz2$'
-    CHECKER_BROWSER_DEFAULT_LANGUAGE = 'en-US'
-    CHECKER_TRANSLATIONS_FILENAME_MATCH = r'.+\.xpi$'
+    FIREFOX_RELEASE_INFO_URL = (
+        "https://product-details.mozilla.org/1.0/firefox_versions.json"  # noqa: E501
+    )
+    FIREFOX_ARCHIVE_BASE_URL = (
+        "https://archive.mozilla.org/pub/firefox/releases/{version}/"  # noqa: E501
+    )
+    FIREFOX_PLATFORM = "linux-x86_64"
+    CHECKER_BROWSER_SOURCE_FILENAME = "firefox.tar.bz2"
+    CHECKER_BROWSER_FILENAME_MATCH = r"firefox-.+\.tar\.bz2$"
+    CHECKER_BROWSER_DEFAULT_LANGUAGE = "en-US"
+    CHECKER_TRANSLATIONS_FILENAME_MATCH = r".+\.xpi$"
 
     def _should_check(self, module_data):
-        return module_data.checker_data.get('type') == 'firefox'
+        return module_data.checker_data.get("type") == "firefox"
 
     def check_module(self, module_data, external_data):
         if not self._should_check(module_data):
             return
 
-        log.debug('Retrieving latest available firefox version')
+        log.debug("Retrieving latest available firefox version")
         # This should raise an error if either the release info url or
         # the returned json data is broken. Lets not catch it here as
         # we want that checker to break if that happens.
         latest_version = self._get_latest_available_version()
         assert latest_version is not None
 
-        log.debug('Latest available firefox version: %s', latest_version)
+        log.debug("Latest available firefox version: %s", latest_version)
 
-        log.debug('Retrieving latest firefox version info')
+        log.debug("Retrieving latest firefox version info")
         latest_info = self._get_latest_version_info(latest_version)
 
-        log.debug('Checking external data')
+        log.debug("Checking external data")
         browser_data = None
         for data in external_data:
             if re.match(self.CHECKER_BROWSER_SOURCE_FILENAME, data.filename):
                 browser_data = data
                 break
 
         if not browser_data:
-            log.warning('Unable to find browser source with filename %s',
-                        module_data.name,
-                        self.CHECKER_BROWSER_SOURCE_FILENAME)
+            log.warning(
+                "Unable to find browser source with filename %s",
```

Comment:
The reformatting here is correct but it has shown that the log message is wrong – it only has one `%s` placeholder.

Reply:
I might be wrong but original code seems to have only one placeholder as well.

Reply:
Yes, I know. Up to you if you fancy fixing it.

Reply:
Ah, sure, misunderstood your comment.